app/src/main/java/com/napzak/market/presentation/home/HomeScreen.kt

```diff
@@ -0,0 +1,160 @@
+package com.napzak.market.presentation.home
+
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.LocalOverscrollConfiguration
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.napzak.market.core.common.state.UiState
+import com.napzak.market.core.designsystem.component.topbar.NapzakLogoTopBar
+import com.napzak.market.core.designsystem.theme.NapzakMarketTheme
+import com.napzak.market.presentation.home.component.HomeBannerPager
+import com.napzak.market.presentation.home.component.HomeMostSearchedItemGroup
+import com.napzak.market.presentation.home.component.HomeRecommendationItemGroup
+import com.napzak.market.presentation.home.state.HomeUiState
+import kotlinx.collections.immutable.toImmutableList
+
+@Composable
+fun HomeRoute(
+    modifier: Modifier = Modifier,
+    viewModel: HomeViewModel = hiltViewModel(),
+) {
+    val uiState by viewModel.uiState.collectAsStateWithLifecycle()
+
+    LaunchedEffect(Unit) {
+        with(viewModel) {
+            getBannerImages()
+            getRecommendedItems()
+            getPopularItems()
+            getMostSearchedItems()
```

Comment:
```suggestion
       viewModel.fetchHomeData()
```
p2: 이렇게 묶어보시는 건 어떨까하고 제안해봅니다!

Reply:
해당 부분은 서버 통신 구현 시에 수정해보도록 하겠습니다!